src/main/java/baritone/process/FarmProcess.java

```diff
@@ -181,6 +203,12 @@ public PathingCommand onTick(boolean calcFailed, boolean isSafeToCancel) {
         List<BlockPos> bonemealable = new ArrayList<>();
         List<BlockPos> openSoulsand = new ArrayList<>();
         for (BlockPos pos : locations) {
+
+            //check if outside of range
+            if (_range != 0 && pos.getDistance(center.getX(),center.getY(),center.getZ()) > _range){
```

Comment:
formatting, put a space after each comma here, and a space before the curly bracket